### PR TITLE
Add SBMTD Mobility Pass vars to Terraform

### LINKED ISSUE #1777: Add SBMTD Mobility Pass variables to Azure config
Following up to #1771, see a review comment that came in too late: https://github.com/cal-itp/benefits/pull/1771#discussion_r1387255135

We don't have a way to configure the SBMTD Mobility Pass values because they aren't backed by Key Vault secrets. 

E.g. we need similar terraform entries as for Courtesy Cards: https://github.com/cal-itp/benefits/blob/dev/terraform/app_service.tf#L124

## Acceptance Criteria

- [x] Each of the `MOBILITY_PASS_*` environment variables referenced in the [data migrations](https://github.com/cal-itp/benefits/blob/dev/benefits/core/migrations/0002_data.py#L236) is backed by a corresponding App Setting via terraform

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #1777

### CODE REVIEW

**Review by thekaveman — COMMENTED** [review]
Can you please confirm these are all added and configured in the 3 different environments (`dev`, `test`, `prod` Key Vaults in Azure?)

**Comment by machikoyasuda** [comment]
Secrets status:
- Dev: Added all new variables for SBMTD, including FrontDoor URL for SBMTD Eligibility Server (Dev), added `mst-server-public-key-url`. Set `mobility-pass-verifier-active` to `True`. Removed `sever-public-key-url`.
- Test and Prod: Added all new variables for SBMTD, added `mst-server-public-key-url`, does not have FrontDoor URLs for Test/Prod envs for Eligibility Server. Set `mobility-pass-verifier-active` to `False`. Did NOT remove `sever-public-key-url` b/c it is still being used.

**Comment by machikoyasuda** [comment]
@thekaveman @angela-tran Need to get this PR re-reviewed, before I can continue set up of SBMTD Mob Pass on Dev.

File: terraform/app_service.tf [hunk 1]
```diff
@@ -157,6 +157,14 @@ resource "azurerm_linux_web_app" "main" {
     "SBMTD_PAYMENT_PROCESSOR_CARD_TOKENIZE_URL"            = "${local.secret_prefix}sbmtd-payment-processor-card-tokenize-url)"
     "SBMTD_PAYMENT_PROCESSOR_CARD_TOKENIZE_FUNC"           = "${local.secret_prefix}sbmtd-payment-processor-card-tokenize-func)"
     "SBMTD_PAYMENT_PROCESSOR_CARD_TOKENIZE_ENV"            = "${local.secret_prefix}sbmtd-payment-processor-card-tokenize-env)"
+    "MOBILITY_PASS_VERIFIER_NAME"                          = "${local.secret_prefix}mobility-pass-verifier-name)"
+    "MOBILITY_PASS_VERIFIER_ACTIVE"                        = "${local.secret_prefix}mobility-pass-verifier-active)"
+    "MOBILITY_PASS_VERIFIER_API_URL"                       = "${local.secret_prefix}mobility-pass-verifier-api-url)"
+    "MOBILITY_PASS_VERIFIER_API_AUTH_HEADER"               = "${local.secret_prefix}mobility-pass-verifier-api-auth-header)"
+    "MOBILITY_PASS_VERIFIER_API_AUTH_KEY"                  = "${local.secret_prefix}mobility-pass-verifier-api-auth-header)"
```
Comment (angela-tran) [inline terraform/app_service.tf:164]:
```suggestion
    "MOBILITY_PASS_VERIFIER_API_AUTH_KEY"                  = "${local.secret_prefix}mobility-pass-verifier-api-auth-key)"
```

**Review by angela-tran — CHANGES_REQUESTED** [review]

**Review by angela-tran — APPROVED** [review]

**Review by thekaveman — APPROVED** [review]

**Pull request merged**